util/Migrator/DbScripts/2023-10-03_00_OrganizationReadOwnerEmailAddresses.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR ALTER PROCEDURE [dbo].[Organization_ReadOwnerEmailAddressesById]
+    @OrganizationId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+    SELECT
```

Comment:
Also consider the same here

Reply:
@cyprain-okeke Updates are in this commit: https://github.com/bitwarden/server/commit/f8354d15b59c6b773ab1848fccb5ecbf7d00755f